kitsune/sumo/static/sumo/less/includes/search-result.less

```diff
@@ -46,14 +46,29 @@
     }
   }
 
+  .question-status {
+    font-size: 13px;
+    margin: 12px 0 0 0;
+
+    .solved {
+      background: transparent @iconsSprite no-repeat scroll -49px -862px;
+      color: rgb(129, 188, 46);
+      display: inline-block;
+      font-family: "Open Sans Light",Arial,Helvetica,sans-serif;
```

Comment:
I literally copy and pasted that from elsewhere in the CSS!! Haha but I will fix. And whatever other instance I copied from. At least they are all alphabetized this time ;)